metadata_validation_conversion/validation/get_ontology_text_async.py

```diff
@@ -38,6 +38,7 @@ def collect_ids(records, core_name=None, module_name=None):
             ids.add(parse_record(value))
         for _, value in record['custom'].items():
             ids.add(parse_record(value))
+    ids = {el.replace(":", "_") if el is not None else el for el in ids}
```

Comment:
for large datasets submission this can lead to OOM errors, we would rather need to have this change before every API request to the OLS server